examples/serenity/voice/src/main.rs

```diff
@@ -53,7 +53,7 @@ async fn main() {
 
     let framework = StandardFramework::new()
         .configure(|c| c
-                   .prefix("~"))
+                   .prefix("!"))
```

Comment:
Why the change in prefix? All of Serenity's examples use the tilde (`~`) as a prefix, and `!` is too commonly used. This is going to cause conflicts with other bots if a user were to run this example mostly unchanged.

Reply:
it doesnt have to be ! it just seemed weird since ~ is already the default